speech/api/speech_gcs.py

```diff
@@ -77,16 +75,15 @@ def _gcs_uri(text):
 
 
 PROTO_URL = ('https://github.com/googleapis/googleapis/blob/master/'
-             'google/cloud/speech/v1/cloud_speech.proto')
+             'google/cloud/speech/v1beta1/cloud_speech.proto')
 if __name__ == '__main__':
     parser = argparse.ArgumentParser()
     parser.add_argument('input_uri', type=_gcs_uri)
-    parser.add_argument('output_uri', type=_gcs_uri)
```

Comment:
The main point of this sample was to demonstrate having an output uri on gcs (thus '_gcs' in the filename). Is that not supported anymore?